scripts/gtcadis.py

```diff
@@ -63,21 +63,7 @@ def setup():
     print('File "{}" has been written'.format(config_filename))
     print('Fill out the fields in these filse then run ">> gtcadis.py step1"')
 
-def step1():
-    config = ConfigParser.ConfigParser()
-    config.read(config_filename)
-
-    hdf5 = config.get('step1', 'geom_file') 
-    cells = [config.get('step1', 'src_cell')]
-    print ('cells', len(cells))
-    xmesh = config.get('step1', 'xmesh').split(',')
-    ymesh = config.get('step1', 'ymesh').split(',')
-    zmesh = config.get('step1', 'zmesh').split(',')
-    #intensities = [config.get('step1', 'intensities')]
-    #print (len(intensities))
-    src_vol = [config.getfloat('step1', 'src_vol')]
-    print ('src vol', len(src_vol))
-    
+def names_dict():
```

Comment:
names_dict and cards will be private functions, so should be preceded by an underscore (e.g. _names_dict)

Reply:
should the function cards also be preceded by an underscore?